docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/how-to-explicitly-implement-members-of-two-interfaces_1.cs

```diff
@@ -19,44 +19,32 @@ class Box : IEnglishDimensions, IMetricDimensions
         float lengthInches;
         float widthInches;
 
-        public Box(float length, float width)
+        public Box(float lengthInches, float widthInches)
```

Comment:
There's a neat trick with Tuples you can use here:

```csharp
public Box(float lengthInches, float widthInches) => (this.lengthInInches, this.widhtInInches) = (lengthInches, widthInches);
```

Reply:
I'm aware of that syntax, but I don't like it much: while it means the code has less lines, it doesn't actually make it shorter. And I think it makes it harder to understand what value is assigned to what field, because those two are further apart.

Should I still go ahead and use that syntax here?

Reply:
😆 I realize it's a bit controversial. I'll merge as you've built it.

It's probably good to have examples of both and see what the larger community thinks.